x/gasless/expected/keeper.go

```diff
@@ -0,0 +1,21 @@
+package expected
```

Comment:
I suggest moving this code to `types/expected_keepers.go` so it's more consistent with the existing code